config/mainCfg_TauTau_Legacy2018.cfg

```diff
@@ -1,42 +1,47 @@
 
-#variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, bjet1_pt, bjet2_pt, tauH_mass, tauH_SVFIT_mass, ditau_deltaR, tauH_pt, bH_pt, bH_mass, HH_pt, HH_mass_raw
-variables = tauH_pt
+# TOGETHER WITH THE TARGET VARIABLES YOU NEED ALWAYS PLOT LOWER LEVEL VARIABLES TO CHECK THAT EVERYTHING IS OK
+variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, ditau_deltaR, bjet1_pt, bjet1_bID_deepFlavor, bjet2_pt, bjet2_bID_deepFlavor, tauH_mass, tauH_SVFIT_mass, tauH_pt, bH_mass, bH_pt, HH_mass_raw, HH_pt, BDToutSM_kl_1
 
-selections = baseline, s1b1jresolved, s2b0jresolved, sboostedLL
+# SELECTIONS W/O MASS CUT
+selections = baseline, s1b1jresolved, s2b0jresolved, sboostedLL, s0b2j, VBFloose, VBFtight
 
-regions    = SR, SStight, SSrlx, OSinviso, SSinviso
+# SELECTIONS W/ MASS CUT
+#selections = baselineMcut, s1b1jresolvedMcut, s2b0jresolvedMcut, sboostedLLMcut, s0b2j, VBFlooseMcut, VBFtightMcut
+
+regions    = SR, SStight, OSrlx, SSrlx, OSinviso, SSinviso
 
 
 [configs]
 
 sampleCfg = config/sampleCfg_Legacy2018.cfg
 cutCfg    = config/selectionCfg_TauTau_Legacy2018.cfg
 
+
 [merge]
 TT        = TTfullyHad, TTfullyLep, TTsemiLep
-
-#WJets     = WJets_HT_0_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
-W     = WJets_HT_0_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
-
-#other	  = TWtop, TWantitop, singleTop_top, singleTop_antitop, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, EWKZ2Jets_ZToNuNu, WW, WZ, ZZ, ZH_HBB_ZLL, ZH_HTauTau, ggHTauTau, VBFHTauTau, WplusHTauTau, WminusHTauTau, ttHJetTononBB, ttHJetToBB, WWW, WWZ, WZZ, TTWJetsToLNu, TTWJetsToQQ, TTWW
-EWK     = EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, EWKZ2Jets_ZToNuNu
+WJets     = WJets_HT_0_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
+EWK     = EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL
 singleT = TWtop, TWantitop, singleTop_top, singleTop_antitop
-H       = ZH_HBB_ZLL, ZH_HTauTau, ggHTauTau, VBFHTauTau, WplusHTauTau, WminusHTauTau, ttHJetTononBB, ttHJetToBB
-TTV     = TTWJetsToLNu, TTWJetsToQQ
-TTVV    = TTWW
-VV      = WW, WZ, ZZ
-VVV     = WWW, WWZ, WZZ
+VH = ZH_HBB_ZLL, ZH_HTauTau, WplusHTauTau, WminusHTauTau
+ttH = ttHJetTononBB, ttHJetToBB
+doubleTsingleV     = TTWJetsToLNu, TTWJetsToQQ, TTZToLLNuNu
+doubleTVV    = TTWW, TTWZ, TTZZ
+doubleV      = WWTo2L2Nu, WWTo4Q, WWToLNuQQ, WZTo1L1Nu2Q, WZTo1L3Nu, WZTo2L2Q, WZTo3LNu, ZZTo2L2Nu, ZZTo2L2Q, ZZTo2Q2Nu, ZZTo4L
+tripleV     = WWW, WWZ, WZZ, ZZZ
 
```

Comment:
It doesn't matter if you merge them here or in the plotter, but:
1. please cross check that the final merging is the same
2. add a comment for others to understand how are they merged in the end (i.e. write a comment to indicate which of these ones go into "others" etc)